covid_model/model_fit.py

```diff
@@ -102,6 +100,21 @@ def run(self, engine, method='curve_fit', window_size=14, look_back=None,
                 model.tags['run_type'] = 'intermediate-fit'
                 model.write_to_db(engine)
 
+            # simulate the model and save a picture of the output
```

Comment:
optionally produce a plot of hospitlizations using the fit so far. alongside the fitted tc's